### PR TITLE
Add email newsletter design tool

### PR DESCRIPTION
**Pull request by adrianthomas**
Newsletters are the indie web's answer to social media - Mail Designer is playing a part in that movement.

Disclaimer: I work on Mail Designer 365, so I hope that's ok 👍

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -248,6 +248,11 @@ <h2><a href="https://www.pixelmator.com/pro/">Pixelmator</a></h2>
             <em>by <a href="https://www.pixelmator.com/blog/">Pixelmator Team</a></em>
             <p>An image editor for Mac and iOS designed to make the most powerful professional image editing tools accessible to everyone</p>
           </li>
+          <li>
+            <h2><a href="https://www.maildesigner365.com/">Mail Designer 365</a></h2>
+            <em>by <a href="https://www.pixelmator.com/blog/">TOWER ONE GmbH</a></em>
```
Comment (splorp) [inline index.html]: This URL was copied and pasted from the previous entry. It should be:

`<em>by <a href="https://tower-one.net/">TOWER ONE GmbH</a></em>`

Comment (adrianthomas) [reply index.html]: Doh, thanks I've fixed it in https://github.com/good/openweb/pull/42/commits/1b44ffa7b01c1bc03f3cadeee420fbc22d370ef1

**Comment by good** [comment]
Thanks a bunch! I added rapidweaver back in, not sure how it got deleted as a part of this PR, but all is well now.

**Comment by adrianthomas** [comment]
Thanks! 
I really shouldn’t do this stuff via the web UI, apologies for the extra work.

**Comment by good** [comment]
No worries at all. I appreciate the contribution.

**Pull request merged**